src/main/java/com/zendesk/maxwell/schema/SchemaStore.java

```diff
@@ -229,8 +266,79 @@ public static SchemaStore restore(Connection connection, MaxwellContext context)
 		return s;
 	}
 
-	private void restoreFrom(Connection conn, BinlogPosition targetPosition) throws SQLException, InvalidSchemaError {
-		PreparedStatement p;
+	private List<ResolvedSchemaChange> parseDeltas(String json) throws IOException {
+		if ( json == null )
+			return null;
+
+		return mapper.readerFor(listOfResolvedSchemaChangeType).readValue(json.getBytes());
+	}
+
+	private HashMap<Long, HashMap<String, Object>> buildSchemaMap(Connection conn) throws SQLException {
+		HashMap<Long, HashMap<String, Object>> schemas = new HashMap<>();
+
+		PreparedStatement p = conn.prepareStatement("SELECT * from `schemas` where server_id = ?");
```

Comment:
oh.  I see.  Right now we only have one chain at once, so we won't end up getting data we don't need (generally).  if and when I write something that occasionally snapshots chains, I can do this there.